src/shared/store/reducers/productReducer.ts

```diff
@@ -0,0 +1,77 @@
+import { IProduct } from "shared/models";
+import { PRODUCT_ACTIONS } from "../actions/productActions";
+
+interface ProductReducerState {
+  products: IProduct[];
+  isLoading: boolean;
+  isError: boolean;
+  error: string;
+}
+
+interface ProductAction {
```

Comment:
Remove

Reply:
i dont understand can you send an example?